src/app/storage.js

```diff
@@ -2,7 +2,56 @@
 
 var utils = require('./utils');
 
-function StorageHandler (updateFiles) {
+function Storage (updateFiles) {
+  var EDITOR_SIZE_CACHE_KEY = 'editor-size-cache';
+
+  this.rename = function (originalName, newName) {
+    var content = this.get(originalName);
+    this.set(newName, content);
+    this.remove(originalName);
+  };
+
+  this.remove = function (name) {
+    window.localStorage.removeItem(name);
+  };
+
+  this.setEditorSize = function (size) {
+    this.set(EDITOR_SIZE_CACHE_KEY, size);
+  };
+
+  this.getEditorSize = function () {
+    return this.get(EDITOR_SIZE_CACHE_KEY);
+  };
+
+  this.getFileContent = function (key) {
+    return this.get(utils.fileKey(key));
+  };
+
+  this.exists = function (key) {
+    return !!this.get(key);
+  };
+
+  this.set = function (key, content) {
+    window.localStorage.setItem(key, content);
+  };
+
+  this.get = function (key) {
+    return window.localStorage.getItem(key);
+  };
+
+  this.keys = function () {
+    return Object.keys(window.localStorage);
+  };
+
+  this.loadFile = function (filename, content) {
```

Comment:
It seems that `loadfile` has to be called with an already "encoded" filename, so its parameter should rather be called `key` and actually the function name should not refer to the concept of a "file".

I think it would be better to not expose the plain `get` and `set` methods here and only give access to file names (where this class would handle the encoding via `utils.fileKey`). If you say that this would be a subsequent refactoring step, I'm fine with that.